lib/src/data/report.dart

```diff
@@ -0,0 +1,88 @@
+import "package:btc_market/data.dart";
+
+/// The type of item to be reported.  
+enum ReportType {
+  /// A product being sold
+  product,
+  /// A conversation between buyer and seller
+  conversation,
+  /// A user's profile. This isn't actually being used, 
+  /// but we keep it here in case
+  userProfile,
```

Comment:
```suggestion
  /// A user's profile (unused for now).
  userProfile,
```

I'd say let's get rid of it then, but on the docs side: you want your comments to look like this

```dart
/// A brief sentence describing the thing
/// 
/// More paragraphs, separated by empty 
/// lines, that give more information.
///
/// Try to separate the paragraphs by topic, 
/// like "how to use" vs "how it works". Be sure
/// to use [codeReferences] when they make sense.
```